composer.json

```
@@ -123,6 +123,7 @@
                 "Remove jquery_ui_slider (3210947)": "https://www.drupal.org/files/issues/2022-02-16/3210947-12.patch"
             },
             "drupal/bootstrap_barrio": {
+                "Fix barrio webform required field issue": "https://gist.githubusercontent.com/tadean/09ef9559553c788c5637077462b392dd/raw/1190b7d78843af29e2eae31c4601a51e65e5b352/barrio_webform_required_field_issue.patch",
```

Adding Drupal-org issue number to patch key.
```suggestion
                "Fix barrio webform required field issue (3133643)": "https://gist.githubusercontent.com/tadean/09ef9559553c788c5637077462b392dd/raw/1190b7d78843af29e2eae31c4601a51e65e5b352/barrio_webform_required_field_issue.patch",
```